README.md

```diff
@@ -50,6 +50,8 @@ You will need to use Node.js `Buffer` API compatible, if you are running inside
   digest: <Buffer 0b ee c7 b5 ea 3f 0f db c9 5d 0d d4 7f 3c 5b c2 75 da 8a 33> }
 ```
 
+For encoding and decoding, the [bs58 module](https://github.com/cryptocoinjs/bs58) is recommended.
```

Comment:
@btrask could you elaborate on why would you recommend to encode multihashes with bs58? We do use b58 encoding in the context of IPFS, but that is a convenience, not a convention to all the multihashes